exporter/exporterhelper/batch_sender.go

```diff
@@ -170,6 +181,7 @@ func (bs *batchSender) sendMergeSplitBatch(ctx context.Context, req Request) err
 		}
 		reqs = reqs[1:]
 	} else {
+		defer bs.activeRequests.Add(-1)
```

Comment:
Then this can be removed

Reply:
I don't think so. The increment on line 164 will be missing a corresponding decrement if we remove this line. This applies to case `len(reqs) > 1 && bs.activeBatch.request == nil`, i.e. split requests that don't touch activeBatch. The correct handling should be incrementing activeRequests (line 164) before sending the split requests, and decrement activeRequests after it is done, which is what this line is responsible for.

Reply:
Ignore my previous comment. I looked at the indentation wrong.